bundler/spec/commands/remove_spec.rb

```diff
@@ -13,7 +13,7 @@
     end
   end
 
-  context "when --install flag is specified" do
+  context "when --install flag is specified", :bundler => "<3" do
```

Comment:
Just notice another small inconsistency of missing a space between "<" and "3". You fix it while squashing if you want, otherwise it's no big deal!

Reply:
No problem